crm_action/views/crm_action_view.xml

```diff
@@ -7,17 +7,18 @@
             <field name="arch" type="xml">
                 <form string="Actions">
                     <header>
-                        <button string="Confirm" name="button_confirm" type="object" states="draft" class="oe_highlight"/>
-                        <button string="Set To Draft" name="button_set_to_draft" type="object" states="done"/>
+                        <button string="Mark as Done" name="button_confirm" type="object" states="draft" class="oe_highlight"/>
+                        <button string="Back to Todo" name="button_set_to_draft" type="object" states="done"/>
```

Comment:
back to Todo and Mark as Done are not translated in French. Can you add it to fr.po?